### DISCUSSION TITLE
SOLVED - How can I create a new brand?

### DISCUSSION THREAD

**Discussion by juareznasato** [quasarframework/quasar#15926] [General - Components / Directives / etc]
Quasar default brand:
primary, secondary, accent, dark, positive, negative, info, warning

Is it possible to create a new "tertiary" brand that can be changed programmatically by setCssVar("tertiary", "#26A69A"); ?

**Comment by metalsadman** [comment]
yes make a class using those css variable.
https://quasar.dev/style/color-palette#adding-your-own-colors
https://quasar.dev/style/color-palette#how-it-works
https://quasar.dev/quasar-utils/color-utils

https://codepen.io/metalsadman/pen/OJvKbov
https://codepen.io/metalsadman/pen/rNWbzJM

**Comment by juareznasato** [comment]
@metalsadman, Thanks for you help!

It works, but when I try to change the theme (Light and Dark) the tertiary does't change.

The code:

**app.scss:**

:root {
  --primary-color: #{$primary};
  --tertiary-color: #{$tertiary};
}
.bg-primary-1 {
  background-color: var(--primary-color);
  filter: brightness(150%);
  color: white;
}
.bg-tertiary-1 {
  background-color: var(--tertiary-color);
  filter: brightness(150%);
  color: white;
}

**quasar.variables.scss:**

$primary: #1976d2;
$secondary: #26a69a;
$tertiary: #e7d216;
...

**Component.vue**
`<q-btn` class="bg-primary-1" />
`<q-btn` class="bg-tertiary-1" />

**ChangeTheme.vue**
    changeTheme(key) {
      if (key === DEFAULT) {
        setCssVar("primary", "#1976d2");
        setCssVar("secondary", "#26A69A");
        setCssVar("tertiary", "#26A69A");
      }
      if (key === DARK) {
        setCssVar("primary", "#121212");
        setCssVar("secondary", "#1d1d1d");
        setCssVar("tertiary", "#1d1d1d");
      }

**Comment by juareznasato** [comment]
I solved this way:
It's necessary to use "q-" prefix to works

**quasar.variables.scss**
$tertiary: #e22121;

**app.scss**
:root {
  --q-tertiary: #{$tertiary};
}
.tertiary {
  background-color: var(--q-tertiary);
  color: var(--q-tertiary);
}


**ChangeTheme.vue**
changeTheme(key) {
if (key === DEFAULT) {
setCssVar("tertiary", "#26A69A");
}
if (key === DARK) {
setCssVar("tertiary", "#1d1d1d");
}
